src/chapters/forms/radio-input.rst

```diff
@@ -0,0 +1,56 @@
+Radio Input
+===========
+
+Radio inputs allow a user to pick one option out of a grouping of options.
+Radio inputs with the same name are grouped. Only one radio input in a group
+can be chosen at a time. The ``value`` attribute of the chosen radio input will
+be submitted. Radio inputs are best used with ``<label>`` tags.
+
```

Comment:
What about mentioning how to unselecting a radio button as an option, reference 20.5.2.1. lchs workbook.  You are more familiar with the CSharp program then me and I like how you have broken down forms and given a good overview.

Reply:
i did create an option about unselecting.  thanks for the idea!